fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/debug/Commands.java

```diff
@@ -0,0 +1,623 @@
+            plannerRepl.printlnKeyValue("tasks", "show the current state of the task stack");
+        }
+    }
+
+    /**
+     * Continue the specified (or one) number of steps.
+     * {@code step [<number>]} continue execution for the next number of steps.
+     */
+    @AutoService(Command.class)
+    public static class StepCommand implements Command<Event> {
+        @Override
+        public boolean executeCommand(@Nonnull final PlannerRepl plannerRepl,
+                                      @Nonnull final Event event,
+                                      @Nonnull final ParsedLine parsedLine) {
+            final List<String> words = parsedLine.words();
+            final int steps;
+            if (words.size() == 2) {
+                steps = NumberUtils.toInt(words.get(1));
+            } else {
+                steps = 1;
+            }
+
+            if (steps == 0) {
+                plannerRepl.printlnError("usage step [number]");
+                return false;
+            }
+
+            plannerRepl.addInternalBreakPoint(new PlannerRepl.CountingTautologyBreakPoint(steps));
+            return true;
+        }
+
+        @Nonnull
+        @Override
+        public String getCommandToken() {
+            return "STEP";
+        }
+
+        @Override
+        public void printUsage(@Nonnull final PlannerRepl plannerRepl) {
+            plannerRepl.printlnKeyValue("step [<number>]", "continue execution by the number of specified steps (default: 1)");
+        }
+    }
+
+    /**
+     * List out all quantifiers.
+     */
+    @AutoService(Command.class)
+    public static class QunsCommand implements Command<Event> {
+        @Override
+        public boolean executeCommand(@Nonnull final PlannerRepl plannerRepl,
+                                      @Nonnull final Event event,
+                                      @Nonnull final ParsedLine parsedLine) {
+            final State state = plannerRepl.getCurrentState();
+            final List<Integer> ids = Lists.newArrayList(state.getQuantifierCache().asMap().keySet());
+            Collections.sort(ids);
+            for (Integer id : ids) {
+                plannerRepl.printKeyValue("id", "qun" + id + "; ");
+                @Nullable final Quantifier quantifier = state.getQuantifierCache().getIfPresent(id);
+                if (quantifier != null) {
+                    plannerRepl.printKeyValue("kind", quantifier.getShorthand() + "; ");
+                    plannerRepl.printKeyValue("alias", quantifier.getAlias().toString() + "; ");
+                    final ExpressionRef<? extends RelationalExpression> rangesOver = quantifier.getRangesOver();
+                    plannerRepl.printKeyValue("ranges over", plannerRepl.nameForObjectOrNotInCache(rangesOver));
+                }
+                plannerRepl.println();
+            }
+            return false;
+        }
+
+        @Nonnull
+        @Override
+        public String getCommandToken() {
+            return "QUNS";
```

Comment:
Could this be `QUANTIFIERS` spelled out or something like that?

Reply:
this is the command you have to type in on the REPL to show all quantifier objects. I'd rather have it very short as I am using this quite often. The other similar commands comparable to this are `REFS` and `EXPS`